package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "group-income",
   "version": "0.0.1",
-  "contractsVersion": "0.1.6",
+  "contractsVersion": "0.1.7",
```

Comment:
Please set this to `0.2.0` - from now on, until `e2e-protocol` is merged, all PRs must use the same contracts version: `0.2.0` because we have not deployed them yet. This will keep us from conflicting with the version on the live server and will prevent us from unnecessarily filling up the repo.